src/Controls/src/Core/Handlers/Items/ItemsViewHandler.Windows.cs

```diff
@@ -386,7 +388,30 @@ FrameworkElement RealizeEmptyView(View view)
 			var handler = view.ToHandler(MauiContext);
 			var platformView = handler.ContainerView ?? handler.PlatformView;
 
-			return platformView as FrameworkElement;
+			if (platformView == null)
+			{
+				return null;
+			}
+
+			platformView.HorizontalAlignment = view.HorizontalOptions.ToCore() switch
+			{
+				Primitives.LayoutAlignment.Fill => HorizontalAlignment.Stretch,
+				Primitives.LayoutAlignment.Start => HorizontalAlignment.Left,
+				Primitives.LayoutAlignment.Center => HorizontalAlignment.Center,
+				Primitives.LayoutAlignment.End => HorizontalAlignment.Right,
+				_ => throw new ArgumentOutOfRangeException()
+			};
```

Comment:
@hartez do we have extension methods for this at all?

Reply:
My IntelliSense didn't provide for an option.